soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11171,7 +11171,21 @@ void Player_DrawGameplay(PlayState* play, Player* this, s32 lod, Gfx* cullDList,
             MATRIX_TOMTX(sp70);
         }
 
-        gSPDisplayList(POLY_OPA_DISP++, sMaskDlists[this->currentMask - 1]);
+       
+        if ((this->currentMask == PLAYER_MASK_BUNNY) && CVarGetInteger("gCosmetics.Equipment_BunnyHood.Changed", 0)) {
+            gDPSetGrayscaleColor(POLY_OPA_DISP++, 75, 75, 75, 255);
+            gSPGrayscale(POLY_OPA_DISP++, true);
+            Color_RGBA8 hoodColor = CVarGetColor("gCosmetics.Equipment_BunnyHood.Value", (Color_RGBA8){ 255, 255, 255, 255 });
+            gDPSetGrayscaleColor(POLY_OPA_DISP++, hoodColor.r, hoodColor.g, hoodColor.b, 255);
+        }
+        if (!((this->currentMask == PLAYER_MASK_BUNNY) && (CVarGetInteger("gHideBunnyHood", 0)))) {
```

Comment:
If it's _not_ bunny hood _or_ hideBunnyHood is off, render it.
```suggestion
        if (this->currentMask != PLAYER_MASK_BUNNY || !CVarGetInteger("gHideBunnyHood", 0)) {
```